### ISSUE TITLE
Very poor docs

### ISSUE THREAD

**Issue by YounesOuterbah** [sarulabs/dingo#29]
very poor docs that could be fixed so new users like me can use it with ease

**Comment by sarulabs** [comment]
Unfortunately, I don't plan to rewrite the whole documentation. But maybe I can help you if you have a problem with a specific part.

**Comment by YounesOuterbah** [comment]
yeah man i would love too, if u can share more examples or proper way to set it up (like from scratch to up n running), or anything helpful
note: i already know how to setup dingo and generate it (learned it from my job codebase) (but not sure if my steps were correct or i messed with step or two)

**Comment by sarulabs** [comment]
I think the best example of the library is the `tests` directory. The structure is close to the one of a real application.

If you are able to retrieve the objects from the generated container, then it works.

The core ideas of the library are probably better explained in the di documentation https://github.com/sarulabs/di. You should read it if you haven't.

**Comment by YounesOuterbah** [comment]
aight man, thanks hope we see more projects and enhancements any time soon
